.github/workflows/UnitTests.yml

```diff
@@ -80,6 +80,10 @@ jobs:
       run: |
         docker run -v /home/runner/actions-runner/_work/maxtext/maxtext:/app --rm --privileged maxtext_base_image bash -c \
         'bash end_to_end/test_decode_checkpoint.sh -r runner_$(date +%Y-%m-%d-%H-%M) -o gs://runner-maxtext-logs -d gs://maxtext-dataset -i 4'
+    - name: Test llama2_decode.py
+      run: |
+        docker run -v /home/runner/actions-runner/_work/maxtext/maxtext:/app --rm --privileged maxtext_base_image bash -c \
+        'python3 MaxText/decode.py MaxText/configs/base.yml run_name=runner_$(date +%Y-%m-%d-%H-%M) base_output_directory=gs://runner-maxtext-logs dataset_path=gs://maxtext-dataset load_from_other_directory=gs://maxtext-llama/llama2-7b/decode-ckpt-scaled-maxtext load_from_other_directory_step=0 per_device_batch_size=1 model_name='llama2-7b' ici_tensor_parallelism=4 steps=1 max_prefill_predict_length=4  max_target_length=16 async_checkpointing=false'
```

Comment:
Can you move your "I love to" overrides here? There aren't really good in base.yml. (Amazing to see this running  :-))

Reply:
(I mean the golden evaluation and the prompt should both be replicated here.)

Reply:
Done - thanks!